clinica/pydra/engine.py

```diff
@@ -52,52 +54,91 @@ def build_input_workflow(pipeline: Workflow, core_workflow: Workflow) -> str:
     """
 
     field = ""
+    readers = []
+    queries = []
 
     list_core_inputs = pu.list_in_fields(core_workflow)
```

Comment:
don't all the fields get added with both calls `list_in_fields` and `list_dict_in_fields`? The condition for adding the elements doesn't seem restrictive